docs/hello_nextflow/09_hello_nf-core.md

```diff
@@ -1,7 +1,258 @@
+    ```
+
+Now that we've got the pipeline pulled, we can try running it!
+
+### Trying out an nf-core pipeline with the test profile
+
+Conveniently, every nf-core pipeline comes with a `test` profile.
+This is a minimal set of configuration settings for the pipeline to run using a small test dataset that is hosted on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository. It's a great way to try out a pipeline at small scale.
+
+The `test` profile for `nf-core/demo` is shown below:
+
+```groovy title="conf/test.config" linenums="1"
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/demo -profile test,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Test profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+    // Input data
+    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
+
+}
+```
+
+This tells us that the `nf-core/demo` `test` profile already specifies the input parameter, so you don't have to provide any input yourself.
+However, the `outdir` parameter is not included in the `test` profile, so you have to add it to the execution command using the `--outdir` flag.
+
+Here, we're also going to specify `-profile docker`, which by nf-core convention enables the use of Docker.
+
+Lets' try it!
+
+```bash
+nextflow run nf-core/demo -profile docker,test --outdir results
+```
+
+Annoyingly, we get the following output:
+
+```console title="Output"
+ N E X T F L O W   ~  version 24.02.0-edge
+
+ ┃ Launching `https://github.com/nf-core/demo` [serene_stallman] DSL2 - revision: 04060b4644 [master]
+
+Downloading plugin nf-schema@2.1.1
+Nextflow version 24.02.0-edge does not match workflow required version: >=24.04.2
+```
+
+Apparently we are using a slightly older version of Nextflow than what the workflow requires.
+
+This is annoying because we might not want to update Nextflow without checking that the update doesn't affect our current work. However, Nextflow makes it up to us by letting us request a specific version on a one-time basis!
+
+You just have to add `NXF_VER=<version>` to the start of your command, like this:
+
+```bash
+NXF_VER=24.09.2-edge nextflow run nf-core/demo -profile docker,test --outdir results
+```
+
+And boom, that gets us past the version mismatch without committing us to any big changes.
```

Comment:
I don't like this detour being such a prominent piece of content. Can we make it a `hint` instead? 
```suggestion
!!! hint "Changing nextflow version"
    Depending on the nextflow version you have installed, this command might fail due to a version mismatch. 
    If that happens, you can temporarily run the pipeline with a different version than you have installed by adding NXF_VER=<version> to the start of your command as shown below:
    
    ```bash
    NXF_VER=24.09.2-edge nextflow run nf-core/demo -profile docker,test --outdir results
    ```
```

Reply:
I'll copy this over to the `dev` branch PR.